jit-compiler/tests/execution.rs

```diff
@@ -67,3 +73,22 @@ fn assigned_functions() {
     let d = compile(input, "main::d");
     assert_eq!(d.call(0), 1);
 }
+
+#[test]
+fn simple_field() {
+    let f = compile(
+        "
+        namespace std::array;
+            let len = 8;
```

Comment:
What does `len` do in here?

Reply:
Is this the degree of the main?

Reply:
This is `std::array::len`. It needs to be there, otherwise it is not found when it's used. The value does not matter because it's a built-in and is overwritten.